lighthouse-core/lib/url-shim.js

```diff
@@ -74,6 +74,18 @@ URL.originsMatch = function originsMatch(urlA, urlB) {
 
 /**
  * @param {string} url
+ * @return {?string}
+ */
+URL.getOrigin = function getOrigin(url) {
```

Comment:
add a test for `getOrigin`? Maybe add whatever is in the network records that don't have valid origins as the bad test case